src/rpc/util.h

```diff
@@ -22,6 +22,8 @@
 #include <variant>
 #include <vector>
 
+static constexpr bool DEFAULT_RPC_DOC_CHECK{false};
```

Comment:
Now that this is off by default, it needs to be enabled in the tests.

See `def write_config`.